telegram/message.py

```diff
@@ -935,6 +935,86 @@ def edit_reply_markup(self, *args, **kwargs):
         return self.bot.edit_message_reply_markup(
             chat_id=self.chat_id, message_id=self.message_id, *args, **kwargs)
 
+    def edit_live_location(self, *args, **kwargs):
+        """Shortcut for::
+
+            bot.edit_message_live_location(chat_id=message.chat_id,
+                                           message_id=message.message_id,
+                                           *args,
+                                           **kwargs)
+
+        Note:
+            You can only edit messages that the bot sent itself (i.e. of the ``bot.send_*`` family
+            of methods) or channel posts, if the bot is an admin in that channel. However, this
+            behaviour is undocumented and might be changed by Telegram.
```

Comment:
While talking about undocumented behaviour: Only one bot can edit a message in a channel, no matter the sender. If a second bot tries it after that, it gets a Bad Request.

Probably too specific for this note, just wanted to mention it.

Reply:
Good to know! But I agree, that it's a bit too specific for this note …